src/components/HomePage/WhatOurStudentsSay.js

```diff
@@ -36,13 +36,13 @@ function WhatOurStudentsSay() {
 				"This program was great! Again, the concepts were explained well and I didn't feel overloaded / overwhelmed with the material, which I think is important, especially for beginners.",
 		},
 		{
-			name: "Hassan Ahmad",
+			name: "Student 3",
 			course: "Python Beginner",
 			reviewText:
 				"The course was really for as the notes were really clear and everyone on the discord was there to help and I would get answers immediately. I look forward to creating projects with these set of skills.",
 		},
 		{
-			name: "Ahmad",
+			name: "Student 4",
 			course: "Python Beginner",
 			reviewText:
 				"The course was really for as the notes were really clear and everyone on the discord was there to help and I would get answers immediately. I look forward to creating projects with these set of skills.",
```

Comment:
prolly wanna get rid of this, its exact the same review as student 3. it was just there for testing purposes

Reply:
done